packages/adminrouter/extra/src/README.md

```diff
@@ -43,7 +43,7 @@ Admin Router allows Marathon tasks to define custom service UI and HTTP endpoint
   }
 ```
 
-In this case `http://<dcos-cluster>/service/service-name` would be forwarded to the host running the task using the first port allocated to the task.
+When your container/task has its own IP (typically when running in a virtual network), `http://<dcos-cluster>/service/service-name` would be forwarded to the container/task's IP using the first port mapping or port definition when USER networking is enabled or the first discovery port otherwise. When your task/container is mapped to the host, it would be forwarded to the host running the task using the first port allocated to the task.
```

Comment:
Actually this is an error on my part in the documentation. The chosen port is defined by the DCOS_SERVICE_PORT_INDEX label, which is a required label.